spec/classes/python_spec.rb

```diff
@@ -262,12 +245,65 @@
             context 'on a Redhat 6 OS' do
               it { is_expected.to contain_class('python::install') }
               it { is_expected.to contain_package('pip').with_name('python-pip') }
+
+              describe 'with python::provider' do
+                context 'scl' do
+                  describe 'with version' do
+                    context '3.6 SCL package' do
+                      let(:params) { { version: 'rh-python36' } }
+
+                      it { is_expected.to compile }
```

Comment:
Can you update it like this:
```suggestion
                      it { is_expected.to compile.with_all_deps }
```